ngmix/tests/test_admom.py

```diff
@@ -66,17 +68,27 @@ def test_admom_smoke(g1_true, g2_true, wcs_g1, wcs_g2):
             jacobian=jac)
         try:
             Tguess = fwhm_to_T(fwhm) + rng.normal()*0.01
-            res = fitter.go(obs=obs, guess=Tguess)
+            res = run_admom(obs=obs, guess=Tguess)
             if res['flags'] == 0:
                 gm = res.get_gmix()
                 _g1, _g2, _T = gm.get_g1g2T()
 
                 g1arr.append(_g1)
                 g2arr.append(_g2)
                 Tarr.append(_T)
-        except GMixRangeError:
+
+                fim = res.make_image()
+                assert fim.shape == im.shape
+
+        except GMixRangeError:  # pragma: no cover
```

Comment:
So for tests, coverage should be complete with no exclusions or we should remove the test code. A test is worthless if we don't execute all of its parts.

Reply:
How do you trigger that?

Reply:
I guess another way to proceed would be to not catch the exception at all.

It is currently never being raised which is why it wasn't covered. But on some other platform it might be raised due to some small difference in floating point operations

Reply:
Or we could take off the pragma and just be happy with lower coverage fraction

Reply:
I think you made a change in Admom itself in this PR that will always catch this error. Thus I think there is no reason to catch it at all in the tests.

Reply:
actually for this particilar test in admom, that can never be raised.  I now am catching it in the admom code.  I'll remove this

Reply:
actually that was to catch the conversion from e1, e2 to g1, g2

I think we can safely ignore it

Reply:
If we can ignore, then let's remove it.